### PR TITLE
Added links to GBIF & BioCASe + correct BioCASe spelling

### PR DESCRIPTION
**Pull request by stijnvanhoey**

### CODE REVIEW

**Comment by peterdesmet** [comment]
Anyway, my comment should not stop this pull request. I will accept it. Thanks @stijnvanhoey

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
 
 The Access to Biological Collections Data (ABCD) Schema is an evolving comprehensive standard for the access to and exchange of data about specimens and observations (a.k.a. primary biodiversity data).
 
-The ABCD Schema attempts to be comprehensive and highly structured, supporting data from a wide variety of databases. It is compatible with several existing data standards. Parallel structures exist so that either (or both) atomised data and free-text can be accommodated. Version 1.2 is currently in use with the GBIF (Global Biodiversity Information Facility) and BioCASE (Biological Collection Access Service for Europe) networks.
+The ABCD Schema attempts to be comprehensive and highly structured, supporting data from a wide variety of databases. It is compatible with several existing data standards. Parallel structures exist so that either (or both) atomised data and free-text can be accommodated. Version 1.2 is currently in use with the [GBIF](http://www.gbif.org) (Global Biodiversity Information Facility) and [BioCASe](http://www.biocase.org) (Biological Collection Access Service for Europe) networks.
```
Comment (debpaul) [inline README.md]: Thanks @peterdesmet for the #tdwg16 presentation on @github!

Comment (MattBlissett) [inline README.md]: GBIF accepts both 1.2 and 2.06 from BioCASe installations.

Comment (peterdesmet) [inline README.md]: @MattBlissett should this be updated in the description?

Comment (MattBlissett) [inline README.md]: I made a pull request! #6 .

**Pull request merged**